### PR TITLE
CHANGELOG for 0.14.0

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+ * "Show references" CodeLens for `#insert` section in Qute templates. See [#814](https://github.com/redhat-developer/quarkus-ls/issues/814).
+ * Support surround with section/comments command for Qute templates. See [#801](https://github.com/redhat-developer/quarkus-ls/pull/801)
+
+### Bug Fixes
+
+ * Treat `CompletionStage` and `Uni` objects as their resolved type for validation and completion in Qute templates. See [#826](https://github.com/redhat-developer/quarkus-ls/issues/826).
+ * `{#if}` sections in Qute templates of the form `{#if !inject:beanName}{/if}` are marked as invalid. See [#828](https://github.com/redhat-developer/quarkus-ls/issues/828).
+ * Quick fix to generate a `@TemplateExtension` class from a Qute template sometimes places the class in the wrong folder. See [#831](https://github.com/redhat-developer/quarkus-ls/issues/831).
+ * Fix ClassCastException in qute template. See [#816](https://github.com/redhat-developer/quarkus-ls/issues/816).
+ * Use quarkus-ls's Qute parser for syntax validation in order to get consistent diagnostic positions and ranges. See [#812](https://github.com/redhat-developer/quarkus-ls/issues/812).
```
Comment (rgrunber) [inline CHANGELOG.md:29]: Could we say "Use the fault tolerant parser ..... " . I assume that would distinguish it as ours ? Also should this be under enhancements maybe ?

**Review by rgrunber — COMMENTED** [review]
Also, should the terms "CodeLens", "CodeAction" be replaced with "Code action", "Code lens" ? Seems like the former are more specific to internals of the API while the latter is more user-friendly.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+ * Improve user tag snippet completion in Qute templates. See [#784](https://github.com/redhat-developer/quarkus-ls/issues/784).
+ * Validate user tag parameters in Qute templates. See [#788](https://github.com/redhat-developer/quarkus-ls/issues/788).
+ * Go to definition for user tag parameters in Qute templates. See [#789](https://github.com/redhat-developer/quarkus-ls/issues/789).
+ * CodeAction to insert required user tag parameters in Qute template. See [#797](https://github.com/redhat-developer/quarkus-ls/issues/797).
+ * Context-aware snippets in Java. See [#782](https://github.com/redhat-developer/quarkus-ls/issues/782).
+ * Snippet, documentation, and validation for Qute #fragment section. See [#768](https://github.com/redhat-developer/quarkus-ls/pull/768).
+ * CodeLens, diagnostics, and document links for Qute fragments. See [#769](https://github.com/redhat-developer/quarkus-ls/pull/769).
+ * When applying the Qute template quick fix to generate a missing class member, generate the member at the end of the class. See [#692](https://github.com/redhat-developer/quarkus-ls/pull/692).
```
[the remark this answers is not in the document]
Comment (datho7561) [reply CHANGELOG.md:18]: I agree this message is a bit wordy, but I think the wording you used implies that when you type out a reference to a missing member, that triggers the member being generated in the class.

File: CHANGELOG.md (hunk 1 above)
Comment (datho7561) [reply CHANGELOG.md:29]: The reason I listed it here was because Angelo made this change to fix a bug where the error range for an errror made no sense and caused a bug in code actions.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## 0.14.0 (March 30, 2023)
+
+### Enhancements
+
+ * Support for uri/uriabs Qute resolvers from Quarkus Renarde. See [#571](https://github.com/redhat-developer/quarkus-ls/issues/571).
+ * Workspace symbols and CodeLens for Renarde application REST endpoints. See [#777](https://github.com/redhat-developer/quarkus-ls/issues/777).
+ * CodeAction to insert expected `<input>` for Renarde `#form` section in Qute template. See [#778](https://github.com/redhat-developer/quarkus-ls/issues/778).
+ * Show validation errors for all Qute templates in a project, even unopened ones, without first needing to open a template. See [#813](https://github.com/redhat-developer/quarkus-ls/issues/813).
```
[the remark this answers is not in the document]
Comment (datho7561) [reply CHANGELOG.md:10]: Okay. The issue linked here is the issue for resolving the "without first needing to open a template" part, that's the only reason I used that wording.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+
+### Enhancements
+
+ * Support for uri/uriabs Qute resolvers from Quarkus Renarde. See [#571](https://github.com/redhat-developer/quarkus-ls/issues/571).
+ * Workspace symbols and CodeLens for Renarde application REST endpoints. See [#777](https://github.com/redhat-developer/quarkus-ls/issues/777).
+ * CodeAction to insert expected `<input>` for Renarde `#form` section in Qute template. See [#778](https://github.com/redhat-developer/quarkus-ls/issues/778).
+ * Show validation errors for all Qute templates in a project, even unopened ones, without first needing to open a template. See [#813](https://github.com/redhat-developer/quarkus-ls/issues/813).
+ * Improve user tag snippet completion in Qute templates. See [#784](https://github.com/redhat-developer/quarkus-ls/issues/784).
+ * Validate user tag parameters in Qute templates. See [#788](https://github.com/redhat-developer/quarkus-ls/issues/788).
+ * Go to definition for user tag parameters in Qute templates. See [#789](https://github.com/redhat-developer/quarkus-ls/issues/789).
```
[the remark this answers is not in the document]
Comment (datho7561) [reply CHANGELOG.md:13]: I don't think combining all three of these is helpful, since it obfuscates what specifically was improved. i.e. a user of qute-ls can't read the changelog and try out the features without first having to read through the linked issues. I'll combine the last two, since they are related.

[the remark this answers is not in the document]
Comment (datho7561) [reply CHANGELOG.md:13]: Come to think of it "improve" isn't a great description of what was actually done, I'll reword that too.

**Comment by angelozerr** [comment]
@datho7561 please add https://github.com/redhat-developer/quarkus-ls/issues/821

**Comment by datho7561** [comment]
@rgrunber is the changelog good to merge?
